src/frontend/apps/desk/src/features/header/AccountDropdown.tsx

```diff
@@ -14,7 +14,12 @@ export const AccountDropdown = () => {
     <DropButton
       button={
         <Box $flex $direction="row" $align="center">
-          <Text $theme="primary">{userName}</Text>
+          <Box $flex $direction="column" $align="left">
+            <Text $theme="primary">{userName}</Text>
+            {userData?.organization?.registration_id_list?.at(0) && (
```

Comment:
I don't understand the condition, why should it be only when there is a registration ID?

Reply:
That does seem indirect now you mention it, testing for the name that we're going to display would work just as well. (I think I was biased by the testing perspective, the distinguishing feature of the organization that satisfies that test is that it's the only one in our test data that has a SIRET.)